apps/site/pages/docs/api-extensions/extending-api-schema.mdx

```diff
@@ -0,0 +1,378 @@
+
+```ts copy filename="graphql/vtex/resolvers/index.ts"
+
+import { default as StoreProductResolver } from './product'
+
+const resolvers = {
+  ...StoreProductResolver,
+}
+
+export default resolvers
+```
+
+Once you have defined these files in the `vtex` folder, the new fields are already available to use. 
+
+<Callout type="warning" emoji="⚠️">
+  If the changes you made still not available, refer to the troubleshooting section [GraphQL changes not visible during development](/docs/api-extension/troubleshooting#graphql-changes-not-visible-during-development).
+</Callout>
+ 
+
+## Extending third-party API schemas
+
+As stores tend to grow, so does the possibility of consuming new data that is not provided by default in the FastStore API or other VTEX APIs. 
+As such, FastStore must consume new data from third-party APIs.
+
+### Step by step
+#### Step 1- Preparing the folders and files
+
+<Callout type="warning" emoji="⚠️">
+  If you already have a directory structure created in the [Extending VTEX API Schemas](/docs/api-extension/extending--api-schema#step-1---preparing-the-folders-and-files),
+  for third-party API schemas you only need to add the `thirdParty` folder inside `src/graphql`.
+</Callout>
+
+1. In you store repository, go to the `src` folder and create the `graphql` subfolder. You can do this by running the following command in your terminal:
+
+<Tabs items={['macOs and Linux', 'Windows']}>
+  <Tab>
+  ```bash copy
+  mkdir graphql
+  ```
+  </Tab>
+
+  <Tab>
+   ```bash copy
+  mkdir graphql
+  ```
+  </Tab>
+</Tabs>
+
+2. Inside the new `graphql` folder, create the `thirdParty` subfolder.
+
+<Tabs items={['macOs and Linux', 'Windows']}>
+  <Tab>
+  ```bash copy
+  mkdir thirdParty
+  ```
+  </Tab>
+
+  <Tab>
+   ```bash copy
+  mkdir thirdParty
+  ```
+  </Tab>
+</Tabs>
+
+3. In `thirdParty`, create `resolvers` and `typeDefs` subfolders.
+<Tabs items={['macOs and Linux', 'Windows']}>
+  <Tab>
+  ```bash copy
+  mkdir resolvers/typeDefs
+  ```
+  </Tab>
+
+  <Tab>
+   ```bash copy
+  mkdir resolvers\typeDefs
+  ```
+  </Tab>
+</Tabs>
+
+<Callout type="warning" emoji="⚠️">
+  The `src/graphql/vtex/resolvers` and `src/graphql/vtex/typeDefs` path is mandatory, but you can organize the files inside those directory as you wish.
+</Callout>
+
+4. Create a `index.ts` file inside the `resolvers` folder.
+
+Once you have created the essentials folders, you will have a folder structure similiar to the the following:
+
+
+```ts copy
+starter.store/
+└── src/
+    └── graphql/
+        ├── thirdParty/
+        │   ├── resolvers/
+        │       └── index.ts
+        │       └── <resolverName>.ts
+        │       └── …
+        │   ├── typeDefs/
+        │       └── <typeName>.graphql
+        │       └── …
+```
+
+<Callout type="info" emoji="ℹ️">
+  This directory structure distinguishes between data 
+  originating from VTEX and data from third-party APIs, which helps in debugging.
```

Comment:
```suggestion
  This directory structure distinguishes between data 
  originating from VTEX and data from third-party APIs.
```

I'd remove this part or complement it further. I don't think developers will find this argument compelling enough, so I'd rather not have it.